src/components/GoogleWalletButton.tsx

```diff
@@ -0,0 +1,207 @@
+
+    const generateSaveUrl = async () => {
+      let genericObject = {
+        id: `${objectId}`,
+        classId: classId,
+        genericType: "GENERIC_TYPE_UNSPECIFIED",
+        hexBackgroundColor: "#1A1A1A",
+        logo: {
+          sourceUri: {
+            uri: "https://i.ibb.co/bbvN62L/logo.png",
+          },
+        },
+        barcode: {
+          type: "QR_CODE",
+          value: "http://nfctap.xyz",
+          alternateText: "nfctap.xyz",
+        },
+        cardTitle: {
+          defaultValue: {
+            language: "en",
+            value: "nfctap.xyz",
+          },
+        },
+        heroImage: {
+          sourceUri: {
+            uri: "https://raw.githubusercontent.com/project-vinyl/nfctap.xyz/579e0cdbda8fab976230deb0594b810f3db747ff/models/sigmoji.pass/strip%403x.png",
+          },
+        },
+        subheader: {
+          defaultValue: {
+            language: "en",
+            value: "Collection",
+          },
+        },
+        header: {
+          defaultValue: {
+            language: "en",
+            value: "Sigmojis",
+          },
+        },
+        textModulesData: [
+          {
+            id: "score",
+            header: "SCORE",
+            body: "?",
+          },
+          {
+            id: "collected",
+            header: "COLLECTED",
+            body: `${number.toString()}/20`,
```

Comment:
This shouldn't be hardcoded, it depends on collection size (which i think is 200 for devconnect)

Reply:
Applies to all of the below fields, lets at least make these constants so its more clear where they are coming from